CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 
 ### Changed
+- Database container images are now pinned to a specific version rather than using `latest`. The tags selected are the most recent as of the time of this change. If a JDBC URL is used with no tag specified, a WARN level log message is output, pending a future change to make tags mandatory in the JDBC URL.
```

Comment:
Please link the PR :)

Reply:
Now that I have one I will :D

Reply:
I have the same issue every time I submit a PR :D Even started to predict the next number 😂